changes/1286-masalim2.md

```diff
@@ -0,0 +1,5 @@
+Raise more informative ``TypeError`` in ``pydantic.utils.ValueItems.__init__``
```

Comment:
single back quotes please, this is markdown. Also I try to keep this to one line (roughly 140 max.) remember there's a link to the actual issue/PR for those who want to understand more.